celer/dense.pyx

```diff
@@ -56,7 +61,10 @@ cdef void set_feature_prios_dense(int n_samples, int n_features, floating[:] the
             prios[j] = 10000
             continue
         Xj_theta = fdot(&n_samples, &theta[0], &inc, &X[0, j], &inc)
-        prios[j] = fabs(fabs(Xj_theta) - 1.) / norms_X_col[j]
+        if positive:
+            prios[j] = fabs(Xj_theta - 1.) / norms_X_col[j]
```

Comment:
@hichamjanati small question, the constraint j on theta is `0. <= Xj_theta <= 1` (as opposed to -1 <= ...<= 1 for the Lasso) ?

Reply:
or rather  `Xj_theta <= 1` ?

Reply:
@mathurinm  yes Xj_theta <= 1